src/cls/_ZPM/PackageManager/Client/REST/PackageManagerClient.cls

```diff
@@ -106,6 +110,60 @@ Method GetModuleManifest(pModuleReference As %ZPM.PackageManager.Core.ResolvedMo
   $$$ThrowStatus($$$ERROR($$$GeneralError,"Download package manifest, something went wrong."))
 }
 
+Method SendAnalyticsToRegistry(action As %String, data As %DynamicObject)
+{
+    Set tRequest = ..GetHttpRequest()
+    Set tRequest.ContentType = "application/json"
+    Do tRequest.SetHeader("INSTALLTOKEN", $Get(^IRIS.TempZPMRInst(data.name,data.version))) 
+    Kill ^IRIS.TempZPMRInst(data.name,data.version)
+    Do data.%ToJSON(tRequest.EntityBody) 
+    Set tSC = tRequest.Post($$$URLENCODE(tRequest.Location_"collect/" _ action))
+    If ($$$ISERR(tSC) || (tRequest.HttpResponse.StatusCode'=200)) {
+        $$$ThrowStatus($$$ERROR($$$GeneralError,"Error Sending Analytics Data"))
+    }
+}
+
+Method SendAnalyticsToGoogle(action As %String, data As %DynamicObject)
+{
+    Set TrackingID = ##class(%ZPM.PackageManager.Settings).GetAnalyticsTrackingId()
+    Quit:(TrackingID = "") 0
+    Set req = ##class(%Net.HttpRequest).%New()
+    Set req.Server = "www.google-analytics.com"
```

Comment:
should not it use SSL?

Reply:
I've added SSL, and fix one more bug invoking nonexistent class